includes/base/element-base.php

```diff
@@ -619,6 +619,47 @@ public static function on_import_replace_dynamic_content( $config, $map_old_new_
 		return $config;
 	}
 
+	/**
+	 * On import replace dynamic content.
+	 *
+	 * @since 3.8.0
+	 * @access public
+	 *
+	 * @param array $config
+	 * @param array $new_ids_map
+	 * @param array|null $controls
+	 *
+	 * @return array Element data.
+	 */
+	public static function updated_on_import_replace_dynamic_content( array $config, array $new_ids_map, $controls = null ) : array {
+		$tags_manager = Plugin::$instance->dynamic_tags;
+
+		if ( empty( $config['settings'] ) || empty( $config['settings'][ $tags_manager::DYNAMIC_SETTING_KEY ] ) ) {
```

Comment:
Are you sure about the first condition? Looks like a mistake...

Reply:
Changed.